api-specificatie/openapi.yaml

```diff
@@ -876,137 +808,154 @@ components:
 
-    adresseerbaarObjectIdentificatie:
+    adresseerbaarobjectidentificatie:
       description: De identificatie van een adresserbaar object uit de BAG. Dit kan een verblijfsobject, een standplaats of een ligplaats zijn.
-      name: adresseerbaarObjectIdentificatie
+      name: adresseerbaarobjectidentificatie
       in: path
       required: true
       schema:
         title: identificatie van een adresseerbaarobject
         type: string
         pattern: '^[0-9]{4}01[0-9]{10}|[0-9]{4}02[0-9]{10}|[0-9]{4}03[0-9]{10}$'
-        example: 1234010000000001
+        example: '1234010000000001'
 
-    pandIdentificatie:
-      description: Identificatie van een pand uit de BAG. Dzeze is 16 cijfers lang.
-      name: pandIdentificatie
+    pandidentificatie:
+      description: Identificatie van een pand uit de BAG. Deze is 16 cijfers lang.
+      name: pandidentificatie
       in: path
       required: true
       schema:
         type: string
         pattern: '^[0-9]{4}10[0-9]{10}$'
-        example: 1234100000000001
+        example: '1234100000000001'
 
-    woonplaatsIdentificatie:
+    woonplaatsidentificatie:
       description: Identificatie van een object uit de BAG. Deze is 4 lang bij een woonplaats en 16 lang bij de andere objecten.
-      name: woonplaatsIdentificatie
+      name: woonplaatsidentificatie
       in: path
       required: true
       schema:
         description: 'Deze wordt gebruikt in de operation, bv. /woonplaatsen/0123. Waarde: 0001-9999.'
         type: string
         pattern: '[0-9]{3}[1-9]|[0-9]{2}[1-9][0-9]|[0-9][1-9][0-9]{2}|[1-9][0-9]{3}'
-        example: 0150
+        example: '0150'
 
-    openbareRuimteIdentificatie:
+    openbareruimteidentificatie:
       description: Identificatie van een object uit de BAG. Deze is 4 lang bij een woonplaats en 16 lang bij de andere objecten.
-      name: openbareRuimteIdentificatie
+      name: openbareruimteidentificatie
       in: path
       required: true
       schema:
         type: string
         pattern: '^[0-9]{4}30[0-9]{10}$'
-        example: 0518300000000001
+        example: '0518300000000001'
 
   schemas:
 
-    KeuzeItem:
+    ZoekResultaat:
       type: object
-      description: 'Dit type bevat een enkel keuze item bestaande uit een weergave tekst en keuze item identificatie'
+      description: 'Dit type bevat een enkel zoekresultaat bestaande uit een weergave tekst en zoekresultaat identificatie'
       properties:
         weergavenaam:
-          description: 'Weergavenaam van de zoeksuggestie'
+          description: 'Weergavenaam van het zoekresultaat'
           title: 'weergavenaam'
           type: string
         identificatie:
+          description: 'Identificatie van het zoekresultaat'
           title: identificatie
           type: string
-        _links:
-          $ref: '#/components/schemas/KeuzeItem_links'
+          
+    ZoekResultaatHal:
+      allOf:
+        - $ref: '#/components/schemas/ZoekResultaat'
+        - properties:
```

Comment:
Hier is type: object weggelaten. Geeft geen foutmelding in swagger

Reply:
Maar is dit fout? Dit is de structuur zoals we die ook bij de Hal componentnen in de BRP gebruiken. Zonder "object" dus.